src/TodoList.js

```diff
@@ -1,28 +1,14 @@
 import React from 'react'
+import TodoListItem from './TodoListItem'
 
-const todoList = [
-    {
-        id: 1,
-        title: 'Study',
-    },
-    {
-        id: 2,
-        title: 'Do Homework',
-    },
-    {
-        id: 3,
-        title: 'Turn in assignment',
-    }
-]
-
-const TodoList = () => (
-    <div>
+const TodoList = ({ todoList }) => (
+    <>
         <ul>
-            {todoList.map(function (item) {
-                return <li key={item.id}>{item.title}</li>
+            {todoList.map((item) => {
```

Comment:
This can be simplified a little since you only have a return statement:

`todoList.map(item => <TodoListItem key={item.id} todo={item} />)`